tslint.json

```diff
@@ -1,31 +1,4 @@
 {
-  // See https://palantir.github.io/tslint/rules/
-  "rules": {
-    // These rules find errors related to TypeScript features.
-    "adjacent-overload-signatures": true,
-    "prefer-for-of": true,
-    "unified-signatures": true,
-    "no-any": true,
-
-    // These rules catch common errors in JS programming or otherwise
-    // confusing constructs that are prone to producing bugs.
-
-    "label-position": true,
-    "no-arg": true,
-    "no-construct": true,
-    "no-duplicate-variable": true,
-
-    "no-invalid-this": true,
-    "no-misused-new": true,
-    "no-shadowed-variable": true,
-    "no-string-throw": true,
-    "no-unused-expression": true,
-    "no-var-keyword": true,
-    "triple-equals": [true, "allow-null-check", "allow-undefined-check"],
-    "typeof-compare": true,
-
-    // These rules make code maintenance easier
-    "no-default-export": true,
-    "prefer-const": true
-  }
+  "$schema": "http://json.schemastore.org/tslint",
+  "extends": ["./node_modules/@loopback/build/config/tslint.common.json"]
```

Comment:
Nice 👍 

I hope we will never run into the situation where `@loopback/build` ends up installed somewhere else in the node_modules hierarchy (e.g. in a parent directory).